Formula/idris2.rb

```diff
@@ -1,31 +1,98 @@
 class Idris2 < Formula
   desc "Pure functional programming language with dependent types"
   homepage "https://www.idris-lang.org/"
-  url "https://github.com/idris-lang/Idris2/archive/v0.5.1.tar.gz"
-  sha256 "da44154f6eba5e22ec5ac64c6ba2c28d2df0a57cf620c5b00c11adb51dbda399"
   license "BSD-3-Clause"
   revision 2
   head "https://github.com/idris-lang/Idris2.git", branch: "main"
 
+  stable do
+    url "https://github.com/idris-lang/Idris2/archive/v0.5.1.tar.gz"
+    sha256 "da44154f6eba5e22ec5ac64c6ba2c28d2df0a57cf620c5b00c11adb51dbda399"
+
+    # Fix loading of dylibs on Monterey. Remove in the next release.
+    # Backport of https://github.com/idris-lang/Idris2/pull/2078
+    on_monterey :or_newer do
+      patch do
+        url "https://github.com/idris-lang/Idris2/commit/36403cbc3dab4ecadd8d4f8c9280ef890de09700.patch?full_index=1"
+        sha256 "c80dd2e96f03eab8446fdcb84d836e3792eb0eb4dbf5e644aec3b8ab4a1eb2f6"
+      end
+      patch :DATA
+    end
```

Comment:
Any reason to not patch unconditionally?

Reply:
Was planning to reduce risk given large number of in-between commit changes. Now that v0.6.0 is merged, this is no longer necessary.